clients/drcachesim/tests/schedule_stats_noise_generator.templatex

```diff
@@ -0,0 +1,5 @@
+Schedule stats tool results:
+Total counts:
+           *[1-9][0-9]* cores
+           2 threads:.*
```

Comment:
The small_trace_file IIRC is asm with a fixed known count of record types, and other tests rely on those?  So should we check that the result has small_trace_file read count plus 100?  Is 100 hidden from us?

Reply:
The `schedule_stats` output does not have a count of record types, it only has a count of instructions.
Right now the noise generator only produces `TRACE_TYPE_READ` records, so that count is unaffected.
There are other metrics printed, such as #idles (not really useful to check for noise) and the context switch histogram (which does have both A and B, where B is the noise generator, but I don't think we can rely on what core B is gonna be scheduled).
The noise generator will generate other records (the read record is just a placeholder for now), so maybe we can revisit this in the future?

Reply:
Run basic_counts at the same time?

Reply:
Done.